articles/quickstart/spa/vuejs/download.md

```diff
@@ -15,7 +15,7 @@ http://localhost:3000
 ```
 4) Make sure [Node.JS LTS](https://nodejs.org/en/download/) is installed and execute the following commands in the sample's directory:
 ```bash
-npm install && npm run serve
+npm install && npm run dev
```

Comment:
This is only `dev` if the user is downloading sample 02 - Calling an API. For 01-Login, `serve` is used.

I think when the download facility is used, it defaults to 01-Login.

Reply:
What do you think about changing the package.json file in the sample "02- Calling an API" like this?
"serve": "npm-run-all --parallel serve backend-dev". 
This way, the content of the download facility works for both examples using 'run serve'. Or should we just leave it like it was?

Reply:
@omar-compres I think that would be a good change for consistency, although the command you've got there won't work.

`"serve": npm-run-all --parallel serve backend-dev"`

See how `serve` just calls itself? You would have to create another script to run the Vue app that does exactly what `serve` does now, then your new command would run that.

Something like:

```
"serve-vue": "vue-cli-service serve",
"serve": "npm-run-all --parallel serve-vue backend-dev"
```

Make sense? If so, can you update the docs here and also raise a PR against [auth0-vue-samples](https://github.com/auth0-samples/auth0-vue-samples) to make this change there too?

Thanks!